sklearn/tests/test_cross_validation.py

```diff
@@ -424,6 +436,11 @@ def test_label_kfold():
     assert_raises(ValueError, cval.LabelKFold, labels, n_folds=3)
 
 
+def test_label_kfold():
+    for shuffle in [False, True]:
+        yield check_label_kfold, shuffle
```

Comment:
We need an additional check that tests that the indices returned when `shuffle` is `True` or `False` are different.